medicaltorch/datasets.py

```diff
@@ -63,27 +65,32 @@ class SegmentationPair2D(object):
 
     :param input_filename: the input filename (supported by nibabel).
     :param gt_filename: the ground-truth filename.
+    :param metadata: metadata related to image
```

Comment:
Mmm that's tricky because... I am not sure that `metadata` will be of interest for `perone/medicaltorch`... And we do not want to move from ivadomed to medicaltorch-neuropoly if at the end we have to move back things from medicaltorch-neuropoly to ivadomed because not relevant to be merged into perone/medicaltorch.
Not sure what is the best way to go..